amqp.go

```diff
@@ -78,7 +83,22 @@ func (amqp *AMQP) Publish(options PublishOptions) error {
 
 	publishing := amqpDriver.Publishing{
 		ContentType: options.ContentType,
-		Body:        []byte(options.Body),
+	}
+
+	if options.ContentType == messagepack {
+		var jsonParsedBody interface{}
+		json.Unmarshal([]byte(options.Body), &jsonParsedBody)
```

Comment:
Delete this...it appears unmarshaling is happening twice.

Reply:
Sorry, absolutely right, it seems that I'm blind... 😅